llvm/lib/Target/EraVM/EraVMTieSelectOperands.cpp

```diff
@@ -86,6 +85,23 @@ bool EraVMTieSelectOperands::tryPlacingTie(MachineInstr &MI,
   if (MI.isRegTiedToUseOperand(0))
     return false;
 
+  // Since COND_OF hasn't reversal version, we have to tie the output register
```

Comment:
Could we phase it more accurately? The logic we implemented guarantee that the 1st argument wound't be tied, but it doesn't necessary mean that the second one would. It might be not a register.
It's also worth showing the example for the normal case scenario - `COND_LT` to explain that normally we reverse condition to optimize if we can place tie to the first argument.

Reply:
Reworked the comments in code. Please review again.